README.md

```diff
@@ -0,0 +1,127 @@
+<a name="readme-top"></a>
+
+<div align="center">
+  
+  <br/>
+
+  <h3><b>Welcome...</b></h3>
+
+</div>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+# 📖 [PORTFOLIO] <a name="about-project"></a>
+
+My PORTFOLIO project in Microverse.
+
+**[PORTFLIO]** is a project experience that Microverse offers to help nuture your skills in designing on small screens.  
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+ 1. HTML5
+ 2. CSS3
+
+
+### Key Features <a name="key-features"></a>
+
+- **[prints out a web page of a portfolio]**
+
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+### Prerequisites
+
+In order to run this project you need:
+
+An extension to your browser
+### Setup
+
+Clone this repository to your desired folder:
+
+git clone [https://github.com/2004-okari/MY-PORTFOLIO](https://github.com/2004-okari/MY-PORTFOLIO)
+### Install
+
+Install this project with:
+
+cd MY-PORTFOLIO
+gem install
+### Usage
+
+To run the project, execute the following command:
+
+Open the index.html file on the browser or use a VS code extension server to start the project`
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **2004-okari**
+
+- GitHub: https://github.com/2004-okari
+- LinkedIn: (https://www.linkedin.com/in/rooney-okari-86a5ba250/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **[Login space]**
+
```

Comment:
- Hey, this is your portfolio project, and from the project milestone, you won't be adding a login feature. kindly check the project requirement to see a bigger picture of what your feature features will be. see the image below to update this section of your readme:

![image](https://github.com/2004-okari/MY-PORTFOLIO/assets/59397159/130675e3-e20a-4da5-ad27-4717e3f5016c)